modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyTree.jsx

```diff
@@ -135,24 +193,54 @@ function VocabularyTree(props) {
         focused={focused}
         onError={onError}
         knownHasChildren={!!hasChildren}
+        selectorComponent={selectorComponent}
+        addOption={onAddOption}
+        removeOption={onRemoveOption}
```

Comment:
These are named a bit backwards, the handler prop should be `on<EventName>`, while the name function passed to the prop should reflect what the function does. I think the code would be easier to understand like this
```suggestion
        onTermSelected={addOption}
        onTermUnselected={removeOption}
```